File: cpp/src/scaneightbitint.cpp
#include "scaneightbitint.h"
#include "scanint.h"
#include "impression.h"
#include "erreur.h"
#include "texte.h"

void scaneightbitint(void)
{
	scanint();
	if (curval < 0 || curval > 255)
	{
		interror(curval, "Bad register code", "A register number must be between 0 and 255.\nI changed this one to zero.");
		curval = 0;
	}
}

File: split/scaneightbitint.pas
procedure scaneightbitint;
begin
  scanint;
  if (curval < 0) or (curval > 255) then
  begin
    begin
      if interaction = 3 then;
      printnl(262);
      print(687);
    end;
    begin
      helpptr := 2;
      helpline[1] := 688;
      helpline[0] := 689;
    end;
    interror(curval);
    curval := 0;
  end;
end;
